views/room.hbs

```diff
@@ -11,18 +11,19 @@
  var timer = document.querySelector('#timer')
  var toggle = document.querySelector('#toggleTimer')
 
- socket.on('new-user', function (data) {
-   console.log(data)
-
-   toggle.addEventListener('click', function () {
-     timer.innerHTML = data.currentTime
-     socket.emit('start-timer')
```

Comment:
This was done this way because eventually each client will be assigned an id and then the `start-timer` event will be emitted with the id. This may not be necessary, I realize but I am still not sure if flattening is a good idea.